arreglo/src/main/java/com/globant/model/Item.java

```diff
@@ -0,0 +1,51 @@
+package com.globant.model;
+
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "items")
+public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    private String name;
+    @ManyToOne
+    @JoinColumn(name = "order_id")
+    private Order order;
+
+
+    public Item(Integer id, String name) {
```

Comment:
remove all geters and setters use the @Data from lombok and use the @NoArgsConstructor & @AllArgsConstructor aswell